glossary.asciidoc

```diff
@@ -109,14 +109,10 @@ Frontier::
   The initial test development stage of Ethereum, lasted from July 30th 2015 to March 2016.
 
 Ganache::
-  A personal blockchain for Ethereum development. One can use Ganache to deploy contracts, develop applications, and run tests. It is available as a desktop application for Windows, Mac, and Linux.
+  personal Ethereum blockchain which you can use to run tests, execute commands, and inspect state while controlling how the chain operates.
+  it is an extremely fast tool for executing transactions and automated testing of smart contracts.
```

Comment:
Capitalization and syntax. This sentence is hanging, without a capitalized first letter. Was it meant to follow a comma, or as a new sentence?